trillian/ctfe/ct_server/main.go

```diff
@@ -123,7 +122,11 @@ func main() {
 		}()
 	} else {
 		// Use a fixed endpoint resolution that just returns the addresses configured on the command line.
-		res = util.FixedBackendResolver{}
+		r, err := naming.NewDNSResolverWithFreq(time.Second)
```

Comment:
Will this work in all cases or should it be optional?

Reply:
I think it's fine hard-coded to this; should work for all cases, and if for some reason somebody needs to resolve less frequently it's easy enough to add a flag for this later on.